lib/ts/recipe/authRecipe/components/feature/authPage/authPage.tsx

```diff
@@ -368,6 +372,11 @@ async function buildAndSetChildProps(
                 (!SuperTokens.usesDynamicLoginMethods || loadedDynamicLoginMethods!.thirdparty.providers.length === 0)
             ) {
                 firstFactors = firstFactors.filter((f) => f !== FactorIds.THIRDPARTY);
```

Comment:
the user was seeing a different error message like frontend doesn't cover all factors or something. I think wherever that message is, that message can also be improved to state exactly which factors are coming from the backend, and which ones are on the frontend, and which ones are missing?

Reply:
That case should be caught earlier by what I added (so we can show a more specific error message), but I also tried to extend the existing one with what the first factors are, where we got the list from and what we have available.